docs/SDK/Explanation/authentication/sessionSigs/use-cases.md

```diff
@@ -0,0 +1,21 @@
+# Use Cases
+
+Below are some use cases of how to use session signatures and capability objects.
+
+_Note that these are not yet currently possible as we need to implement proof verification logic for delegation chains._
```

Comment:
might be awk to highlight these use cases if they aren't yet possible but open to what other's think

Reply:
I think this fits more within a post as a possibility with capabilities and session sigs!

I'm happy with keeping this page in the drafts until it's possible to do.

Reply:
okay, just going to remove it in sidebars